### ISSUE TITLE
Feedback data project

### ISSUE THREAD

**Issue by rodvgomez** [NumEconCopenhagen/projects-2023-aml#4]
Some recommendations: 

About the chapter `Explore each data set`:

1.- Graph: I would recommend expressing the investment and saving using logarithms to smooth out the series. 
2.- Be careful about which series you are using. By definition, savings are equal to investment (Macroeconomic identity). Nevertheless, your graph is showing a gap between both series. This could be because you are using a specific kind of investment.  Just be clear in defining what series you are employing. 
3.- I think it is unnecessary to show the data table, the purpose of using a graph is precisely that, not to show tables.

About the chapter `Analysis`:
Basically, in the analysis you showed the same graph and a new graph of the fourth difference but using the original series and not the logarithmic series. The problem with that is that, instead of calculating the annual growth, you are calculating the growth of growth of the growth of the quarterly growth. So, be careful with that because in terms of economic analysis it does not say something relevant.
